sdk/resources/arm-resources/src/lro/index.ts

```diff
@@ -0,0 +1,2 @@
+export { shouldDeserializeLRO } from "./requestUtils";
+export { LROPoller } from "./lroPoller";
```

Comment:
I finished most of the work needed to move those lro files to `core-lro` here https://github.com/Azure/azure-sdk-for-js/pull/15949 and I believe it should be merged before you GA this package.

Reply:
@deyaaeldeen When the work can be finished and corresponding codegen can be released? We are going to release packages for 4 RPs in preview from this week. Will your work be included it?

Reply:
We agreed offline to target the second week of August.